### ISSUE TITLE
triggering reload event from cli

### ISSUE THREAD

**Issue by igregson** [BrowserSync/browser-sync#405]
Greetings,

Is is possible to trigger the reload event from Browser Sync's cli? Essentially, I'm trying to use use Browsersync as an npm script. So far so good, only I need to be able to trigger it to reload after other tasks have finished.

**Comment by shakyShane** [comment]
If you could provide a sample project showing your desired setup, I'd be happy to have a look at supporting this.

**Comment by igregson** [comment]
Apologies for dropping the ball on this... I changed approaches and no longer needed this. 

But I was happy to just now see that it appears to have been added (according to [the Browser Sync command line docs](http://www.browsersync.io/docs/command-line/). 

While I can't point to a specific use case, I was attempting this while trying to move some projects from Gulp to NPM, as inspired by this post, [How to Use NPM as a Build Tool](http://blog.keithcirkel.co.uk/how-to-use-npm-as-a-build-tool/).

Anyways, thanks again for creating Browser Sync :)
